lib/redshift_serverless_namespace_stack.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
         namespace_configuration = {
             "namespace_name": namespace_name,
             "admin_username": REDSHIFT_DEFAULT_USER,
-            "admin_user_password": secret.secret_value,
+            "admin_user_password": secret.secret_value.unsafe_plain_text(),
```

Comment:
Hi @bwighane, let me look into this and understand the necessary implications before merging.

Reply:
Hey @BranfordTGbieor that works.